lib/controller/controller.go

```diff
@@ -1085,3 +1085,12 @@ func (ctrl *ProvisionController) getStorageClassFields(name string) (string, map
 func claimToClaimKey(claim *v1.PersistentVolumeClaim) string {
 	return fmt.Sprintf("%s/%s", claim.Namespace, claim.Name)
 }
+
```

Comment:
IMO we should make a util folder and put it in there like lib/util/util.go. Then we can copy over what we need from upstream pkg/volume/util.go (until/unless we refactor upstream to include those in client-go)